src/main/java/com/stuypulse/robot/Constants.java

```diff
@@ -55,4 +81,81 @@ public interface Constraints {
 		double MAX_ANGULAR_VEL = (MAX_VEL * 2) / TRACK_WIDTH_METERS;
 		double MAX_ANGULAR_ACC = (Math.pow(MAX_ACC, 2)) / (TRACK_WIDTH_METERS/2); // a = v^2 / r
 	}
+
+	public interface EdwinSettings {
+        // If speed is below this, use quick turn
+        SmartNumber BASE_TURNING_SPEED = new SmartNumber("Driver Settings/Base Turn Speed", 0.4);
+
+        // Low Pass Filter and deadband for Driver Controls
+        SmartNumber SPEED_DEADBAND = new SmartNumber("Driver Settings/Speed Deadband", 0.05);
+        SmartNumber ANGLE_DEADBAND = new SmartNumber("Driver Settings/Turn Deadband", 0.05);
+
+        SmartNumber SPEED_POWER = new SmartNumber("Driver Settings/Speed Power", 1.0);
+        SmartNumber ANGLE_POWER = new SmartNumber("Driver Settings/Turn Power", 1.0);
```

Comment:
move these out of EdwinSettings.. they can be used to drive any drivetrain (kinda, they are technically tuned per robot)

Reply:
see comment about separate Constant files and arcadedrive / drivetrain command